src/client.rs

```diff
@@ -16,7 +16,9 @@ use futures::stream::{self, StreamExt};
 #[cfg(test)]
 use mockall::*;
 use nostr::Event;
+use nostr_sdk::ClientSigner;
 
+#[allow(clippy::struct_field_names)]
```

Comment:
Clippy: `warning: field name starts with the struct's name` implied by `#[warn(clippy::pedantic)]`